README.md

```diff
@@ -9,6 +9,13 @@ Replicated [Akka Persistence](https://doc.akka.io/docs/akka/current/scala/persis
 
 [![Build Status](https://travis-ci.org/akka/akka-persistence-cassandra.svg?branch=master)](https://travis-ci.org/akka/akka-persistence-cassandra)
 
+## History
+
+This [Apache Cassandra](https://cassandra.apache.org/) plugin to Akka Persistence was initiated originally by Martin Krasser, [@krasserm](https://github.com/krasserm) in 2014.
+
+It moved to the [Akka](https://github.com/akka/) organisation in 2017 and the first release after that move was 0.80 in January 2018.
```

Comment:
Should we add the link to the original repo as well?

Reply:
Added.